setup.py

```diff
@@ -1,9 +1,14 @@
 from setuptools import setup, find_packages
 
+# The version is updated automatically with bumpversion
+# Do not update manually
+__version = "__version="0.2.0""
```

Comment:
This does not seem correct. Looks like a typo. It should be:
`__version = "0.2.0"`

Reply:
Thanks